Makefile

```diff
@@ -78,6 +79,15 @@ $(warning No code-sign identity found (CODE_SIGN_IDENTITY is not set))
 endif
 endif
 
+ifeq ($(TARGET_PLATFORM),win32)
+ifndef CODE_SIGN_CERTIFICATE
+$(warning No code-sign certificate found (CODE_SIGN_CERTIFICATE is not set))
+endif
+ifndef CODE_SIGN_CERTIFICATE_PASSWORD
+$(warning No code-sign certificate password found (CODE_SIGN_CERTIFICATE_PASSWORD is not set))
+endif
```

Comment:
Sounds good to me :) Let me know if I can help with any OS X testing (if you don't have access to OS X yet, you might want to ping the operations team to ship you a Mac Mini or something like that anyway)!